index.js

```diff
@@ -82,6 +82,9 @@ exports.register = function (server, options, next) {
                 }, function(err, result){
 
                     if(err){
+                        if (debug) {
+                            internals.log('error', err.formatted);
```

Comment:
That's quite bizarre. (full disclosure: I work with agiri and helped debug this).

It's certainly available on the object outputted. It's part of Libsass.

Here's the file and line reference

https://github.com/sass/libsass/blob/676383ac8364d574815c83c512cd17a25842c3cd/src/sass_context.cpp#L106

So it would appear that it's a version issue on your end perhaps? Libsass added this in this Jan 17th commit:

https://github.com/sass/libsass/commit/d51706793e0cb2c683710ae4e11dd7c62834eeb1

It's certainly a supported feature in node-sass and libsass